### DISCUSSION TITLE
Sculpfun S6 upgrade to expansion kit

### DISCUSSION THREAD

**Discussion by Mas60022** [arkypita/LaserGRBL#1785] [General]
I have Sculpfun S6 and upgade it with expansion kit from 410x450mm to 410x950 mm but i don't find any information what i need to change in software LaserGRBL to get the correct settings for my Laser ?

**Comment by Tribo-2dot0** [comment]
change the values in $130 and $131

**Comment by StuartB4** [comment]
Just change $131 to 950. (Y axis Maximum Travel.)

**Reply by Mas60022** [reply]
thx for help.
next question is about 132 z-axis is now set to 200mm chould i change it or not when i upgrade to 410x950 ?
